src/Simulation.ts

```diff
@@ -262,14 +279,36 @@ class Simulation {
                 errorEngine = []
             } = this.physicalOptions;
 
-            if (engines.length === this.engines.length) {
+            if (engines.length === 2 || engines.length === 6) {
                 for(let i = 0; i < engines.length; i++){
                     if(engines[i]<=1.0 && engines[i]>= -1.0){
-                        this.engines[i] = engines[i];
 
-                        const errorFunction = errorEngine[i] || (v=>v);
+                        // Setting engines with a length of 2 will be deprecated and is only to keep current algorithms working.
+                        if (engines.length === 2) {
```

Comment:
Error function is not applied in the 2-wheel case.

Reply:
Why do we need six controllable wheels if we cannot change the steering of each wheel individually?